.github/workflows/test.yml

```diff
@@ -12,16 +12,31 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - name: setup python
-      uses: actions/setup-python@v4
+    - name: Cache conda
+      uses: actions/cache@v2
+      env:
+        # Increase this value to reset cache if etc/example-environment.yml has not changed
+        CACHE_NUMBER: 0
       with:
-        cache: pip
-        cache-dependency-path: pyproject.toml
-        python-version: "${{ matrix.python-version}}"
-    - name: upgrade dependencies
-      run: |-
-        python -m pip install --upgrade pip hatch
-        pip install -e.[test]
+        path: ~/conda_pkgs_dir
+        key:
+          ${{ runner.os }}-conda-${{ env.CACHE_NUMBER }}-${{
+          hashFiles('test-environment.yml') }}
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        activate-environment: test-nbconvert-a11y
+        channel-priority: strict
+        auto-activate-base: true
+        environment-file: test-environment.yml
+        use-only-tar-bz2: true 
+        miniforge-variant: Mambaforge
+        use-mamba: true
+    - name: init playwright nbconvert-a11y
+      run: |
+        conda activate test-nbconvert-a11y
+        playwright install --with-deps chrome

```

Comment:
this is an excellent candidate for caching, if it puts its downloaded files in a sensible, predictable place

Reply:
i'll add this to the to do list. i wanna get to this using the discussion in https://github.com/microsoft/playwright/issues/7249